src/libs/antares/study/load.cpp

```diff
@@ -143,11 +143,11 @@ void Study::parameterFiller(const StudyLoadOptions& options)
 
     // calendar update
     if (usedByTheSolver)
-        calendar.reset(parameters, /*force leapyear:*/ false);
+        calendar.reset({parameters.dayOfThe1stJanuary, parameters.firstWeekday, parameters.firstMonthInYear, false});
     else
-        calendar.reset(parameters);
+        calendar.reset({parameters.dayOfThe1stJanuary, parameters.firstWeekday, parameters.firstMonthInYear, parameters.leapYear});
 
-    calendarOutput.reset(parameters);
+    calendarOutput.reset({parameters.dayOfThe1stJanuary, parameters.firstWeekday, parameters.firstMonthInYear, parameters.leapYear});
```

Comment:
Could indeed be passed by `&&`. But then make sure to mark the copy constructor as `delete`d

Reply:
I don't know if this usage should direct the implementation choice of the interface.
For all intent and purpose, parameters could have a Settings struct instead of several values and e would be passing a copy

Reply:
Yes, that would be nice